src/use_i18n.rs

```diff
@@ -1,23 +1,35 @@
-use std::path::PathBuf;
-
 use dioxus_lib::prelude::*;
 use fluent::{FluentArgs, FluentBundle, FluentResource};
-
 use unic_langid::LanguageIdentifier;
 
+use std::collections::HashMap;
+
+#[cfg(not(target_arch = "wasm32"))]
+use std::path::PathBuf;
+
+/// `Locale` is a "place-holder" around what will eventually be a `fluent::FluentBundle`
+#[cfg_attr(test, derive(Debug, PartialEq))]
 pub struct Locale {
     id: LanguageIdentifier,
     resource: LocaleResource,
 }
 
 impl Locale {
+    #[deprecated(
```

Comment:
I don't see why these methods have to be deprecated, I am fine having a Into impl alongside this.